scrapers/RealJamVR.yml

```diff
@@ -9,22 +9,23 @@ xPathScrapers:
     scene:
       Title:
         selector: //h1
-      Date:
-        selector: //div[contains(@class, "date")]
         postProcess:
           - replace:
-              - regex: ','
-                with: ""
-          - parseDate: January 2 2006
+              - regex: ^\s+(.+)\s+$
+                with: $1
+      Date:
+        selector: //div[@class="specs-icon"]/following-sibling::strong
+        postProcess:
+          - parseDate: January 2, 2006
```

Comment:
This is returning dates like `Jan. 2, 2021` for [this scene](https://realjamvr.com/scene/realjamcasting-mackenzie-mace/) but it works if you replace the parseDate string with `Jan. 2, 2006`

Reply:
but they used full month names in scenes like [this](https://realjamvr.com/scene/your-lustful-little-stepsister-2/). 
So... `January` becomes `Jan.` but `July` stays `July` ...? weird...
Would you're suggestion for `parseDate` take this in consideration? 
I'll try it out soon ;)
Thanks for your help!

Reply:
Sure, let's throw another regex at it 😁 
```
  Date:
    selector: //div[@class="specs-icon"]/following-sibling::strong
    postProcess:
      - replace:
          - regex: ^([a-zA-Z]{3})\D*(\d{1,2},\s*\d+)$
            with: $1. $2
      - parseDate: Jan. 2, 2006
```

Example results
| Scene                                                             | Before        | After      |
| ----------------------------------------------------------------- | ------------- | ---------- |
| https://realjamvr.com/scene/realjamcasting-mackenzie-mace/        | Jan. 2, 2021  | 2021-01-02 |
| https://realjamvr.com/scene/independent-audit-by-lauren-phillips/ | 2023-03-30    | 2023-03-30 |
| https://realjamvr.com/scene/your-best-friends-daughter/           | 2023-04-27    | 2023-04-27 |
| https://realjamvr.com/scene/your-lustful-little-stepsister-2/     | 2023-07-30    | 2023-07-30 |
| https://realjamvr.com/scene/sex-fitball-workout/                  | Nov. 17, 2016 | 2016-11-17 |

Reply:
updated ;)